### DISCUSSION TITLE
Unable to download Eclipse Birt on mac

### DISCUSSION THREAD

**Discussion by AmaniGW** [eclipse-birt/birt#1524] [Q&A]
Hi,  I've been trying to download the "birt-report-designer-all-in-one-4.14.0-202312020807-macosx.cocoa.x86_64" or " birt-report-designer-all-in-one-4.14.0-202312020807-macosx.cocoa.aarch6"  on my macbook air with an M1 chip. Both all in one packages for macos give me the error "Eclipse is damaged and can’t be opened. You should move it to the Trash." Any idea on how to get past this?

**Comment by merks** [comment]
Sounds like this

https://github.com/eclipse-birt/birt/issues/1514

**Reply by AmaniGW** [reply]
Thank you merks. When I run the "xattr -d com.apple.quarantine eclipse.app" on the terminal, it says: "xattr: No such file: eclipse.app".

**Reply by merks** [reply]
Did you run this in the parent folder of the app?  If you run ‘ls’ does it show the app name?

**Reply by AmaniGW** [reply]
When I run 'ls' in the terminal, I can see both Eclipse and Eclipse Environment. However, the  "xattr -d com.apple.quarantine eclipse.app"  line still says: "xattr: No such file: eclipse.app". I did not understand your first question though. How can I run the terminal request in the parent folder? Thank you!

**Reply by merks** [reply]
The point is, that if it's spelled Eclipse.app but you typed eclipse.app, then it will fail.  But I don't have a mac so I'm suggesting that you determine exactly the right name, with the right case, and ensure that the name visible in the folder in which you are trying to run the command.

**Reply by jlpeabody** [reply]
I didn't even bother downloading BIRT directly as I couldn't get it to work to save my soul. What does work is to first install the Eclipse IDE and then to use that to install the BIRT toolset. Eclipse can be had via https://www.eclipse.org/downloads/

Run Eclipse and from the Help menu select Install New Software...

The source to install from is https://download.eclipse.org/birt/updates/release/latest or choose the directory that you are specifically interested in if a specific version of BIRT is required. If the current version of Eclipse is being used, then version requirements of foundation software should be met (Java and Eclipse).

This user was generating reports very quickly after following these steps but I also have prior experience with Eclipse, installing various perspectives in Eclipse and a history with Actuate reporting (tho BIRT is much different that Actuate was). I know this is 6 weeks late getting to you and if it doesn't help, I hope someone else will encounter this. I went down your path at first and it was painful -- it didn't need to be.
